web/cobrands/fixmystreet/position_map.js

```diff
@@ -1,21 +1,3 @@
-function position_map_box() {
-    var $html = $('html');
-    if ($html.hasClass('ie6')) {
-        $('#map_box').prependTo('body').css({
-            zIndex: 0, position: 'absolute',
-            top: 0, left: 0, right: 0, bottom: 0,
-            width: '100%', height: $(window).height(),
-            margin: 0
-        });
-    } else {
-        $('#map_box').prependTo('body').css({
-            zIndex: 0, position: 'fixed',
-            top: 0, left: 0, right: 0, bottom: 0,
-            width: '100%', height: '100%',
-            margin: 0
-        });
-    }
-}
-
+function position_map_box() {}
```

Comment:
This has been removed here, can the other position_map functions also similarly be removed? I assume they will be confusing things rather than helping now! Looks to be the following: angus/eastsussex/hart/oxfordshire/warwickshire – certainly the map on Oxfordshire goes 'wrong', appearing under the sidebar again.

And also the map_fix call has been removed, so all the map_fix definitions can be removed too.